### PR TITLE
feat: unit test onboarding

### PR DESCRIPTION
**Pull request by sonle-geekyants**
**- What I did**

Fixes [#694](https://github.com/atsign-foundation/apps/issues/694)

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart [hunk 1]
```diff
@@ -17,11 +17,14 @@ class AtOnboardingAccountsScreen extends StatefulWidget {
...
   final String? newAtsign;
 
+  final OnboardingService onboardingService;
+
   const AtOnboardingAccountsScreen({
     Key? key,
     required this.atsigns,
     this.message,
     this.newAtsign,
+    required this.onboardingService,
```
Comment (nitesh2599) [inline at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart]: @sonle-geekyants We have added a new required parameter here, so, will this be a new major version release ?

File: at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart [hunk 2]
```diff
@@ -23,12 +23,16 @@ class AtOnboardingActivateScreen extends StatefulWidget {
...
   final AtOnboardingConfig config;
+  final OnboardingService onboardingService;
+  final FreeAtsignService freeAtsignService;
 
   const AtOnboardingActivateScreen({
     Key? key,
     required this.hideReferences,
     this.atSign,
     required this.config,
+    required this.onboardingService,
```
Comment (nitesh2599) [inline at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart]: New required params, will this impact any existing functionality?

Comment (sonle-geekyants) [reply at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart]: No. I've tested the full functionality, these are working well.

File: at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart (hunk 1 above)
Comment (sonle-geekyants) [reply at_onboarding_flutter/lib/screen/at_onboarding_accounts_screen.dart]: Yes, we need to increase major version

**Comment by gkc** [comment]
@sachins-geekyants Can you review this PR please?

**Review by sachins-geekyants — APPROVED** [review]

**Pull request merged**